benchmark/sirun/appsec-iast/insecure-bank.js

```diff
@@ -0,0 +1,14 @@
+const http = require('http')
+const app = require('/opt/insecure-bank-js/app')
+
+const { port } = require('./common')
+
+app.set('port', port)
+const server = http.createServer(app)
+
+function onListening () {
+  server.close()
+}
+
+server.listen(port)
+server.on('listening', onListening)
```

Comment:
```suggestion
server.listen(port, server.close)
```

or too much compressed? :smile:

Reply:
if `listen` callback receives a parameter, unexpected params can go to `server.close` method.
If you want to compress it, please try with something like:

```
server.listen(port, () => { server.close() })
```